src/displayBackend/drm/Connector.cpp

```diff
@@ -91,10 +91,19 @@ void Connector::init(uint32_t width, uint32_t height,
 
 	if (mCrtcId != cInvalidId)
 	{
-		throw Exception("Already initialized", EINVAL);
+		throw
+		 Exception("Already initialized", EINVAL);
```

Comment:
Why do you need this change?

Reply:
> Why do you need this change?

strange, in my editor this is one line

Reply:
Yep. You can check diffs of your commits before publishing them to avoid such unnecessary changes.